src/components/rangeslider/draw.js

```diff
@@ -92,8 +95,13 @@ module.exports = function(gd) {
         opts._height = (fullLayout.height - margin.b - margin.t) * opts.thickness;
         opts._offsetShift = Math.floor(opts.borderwidth / 2);
 
-        var x = margin.l + (graphSize.w * domain[0]),
-            y = fullLayout.height - opts._height - margin.b;
+        var x = margin.l + (graphSize.w * domain[0]);
+
+        var y = (
+            margin.t + graphSize.h * (1 - oppDomain[0]) +
+            tickHeight +
+            opts._offsetShift + constants.extraPad
+        );
```

Comment:
I guess something in here is responsible for the [small position differences in the mocks](https://github.com/plotly/plotly.js/pull/1355/commits/ffccc75aa40d195d49138fade77a2c00ef89ec6b)? I'm not bothered by the differences, *except* that some of the new ones look a little bit fuzzier than the old ones. Can you try and round it so that the outer edges of the range slider border are exactly on a pixel boundary, even if you have non-integers in `layout.height`, the margins, `rangeslider.borderwidth` and anything else that comes into the calculation?